test/functional/is-browser-relevant.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function isBrowserRelevant (testName, browserAlias) {

```

Comment:
For the same reason as the file get-test-error.js is dedicated too. I thought this variant better, than overflowed setup.js file.